CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.21.0 / 2019-10-TBD
```

Comment:
nit: Do you anticipate additional changes before releasing this and or do you know when you might release it yet?
It doesn't really matter if don't, it can be updated later or even retrospectively.

Reply:
Another PR expected to remove the exceptions for specific CCGs. Probably will want to release both PRs together, but not confirmed as yet.